tests/Spout/Writer/ODS/WriterWithStyleTest.php

```diff
@@ -56,42 +66,59 @@ public function dataProviderForInvalidStyle()
         return [
             ['style'],
             [new \stdClass()],
-            [null],
         ];
     }
 
     /**
      * @dataProvider dataProviderForInvalidStyle
-     * @expectedException \Box\Spout\Common\Exception\InvalidArgumentException
      *
      * @param \Box\Spout\Writer\Common\Entity\Style\Style $style
      */
     public function testAddRowWithStyleShouldThrowExceptionIfInvalidStyleGiven($style)
     {
+        if (version_compare(PHP_VERSION, '7.0.0') >= 0) {
```

Comment:
why did you have to add that?

Reply:
Because PHP < 7 does not know what a ```TypeError``` is.  This is one of the things I would like to adress in the future. "How should we handle Type Errors across PHP Versions etc".

Reply:
I guess the best way to handle that would be to only support PHP >= 7 :)

Reply:
We have migrated our product to PHP 7.1.. So I would be full in :D